kubemarine/core/defaults.py

```diff
@@ -171,7 +171,8 @@ def apply_registry(inventory: dict, cluster: KubernetesCluster) -> dict:
             containerd_endpoints = ["%s://%s" % (protocol, registry_mirror_address)]
 
         old_format_result, _ = contains_old_format_properties(inventory)
-        if old_format_result:
+        # todo remove p3_reconfigure_registries after next release
+        if old_format_result or (cluster and not cluster.context.get('p3_reconfigure_registries', True)):
```

Comment:
bool(cluster) is always True. Also I suppose the same check should be applied to all cases where enrichment has changed. I remember that we also changed enrichment in containerd.enrich_inventory

Reply:
bool(cluster) was fixed.
Enrichment changes only for registries, if they are enriched in new format. In other cases nothing changes.